assets/wizards/dashboard/components/brand-header.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Newspack Dashboard, Brand-Header
+ *
+ * Displaying stored logo and header bg color in a header
+ */
+
+import { BoxContrast } from '../../../components/src';
+
+const { settings } = window.newspack_dashboard;
+
+const BrandHeader = () => {
+	return (
+		<header
+			className="newspack-dashboard__brand-header"
+			style={ { backgroundColor: settings.headerBgColor } }
+		>
+			<BoxContrast
+				className="brand-header__inner"
+				content={ <h1>{ settings.siteName }</h1> }
```

Comment:
Not a blocker at all, just wondering: why not just use `children` and nest the content inside `BoxContrast`?

Reply:
That makes more sense - nice suggestion.